conf/messages.cy

```diff
@@ -399,37 +399,37 @@ property.details.cases.table.challenge=Her
 property.dvr.challenge.error=Os ydych chi’n dal i feddwl bod gennych chi’r hawl i wneud Her,   bydd angen i chi <a class="govuk-link" href="{0}"> lawrlwytho a llenwi ffurflen Her (T012)</a>
 
 assessments.table.caption=Prisiadau
-assessments.table.1=Dyddiad dod i rym
-assessments.table.2=Gwerth ardrethol
-assessments.table.3=Capasiti
-assessments.table.4=O
-assessments.table.5=Hyd
-assessments.table.6=Gweithredoedd
-assessments.table.action=Gweithred
+assessments.table.1=Prisiadau
+assessments.table.2=Dyddiad dod i rym
+assessments.table.3=Cysylltiad a r eiddo
+assessments.table.4=Gwerth ardrethol
 assessments.enddate.present=Presennol
 assessments.enddate.historic.empty=31 Mawrth 2023
 assessments.view.valuation=Gweld prisiad
 assessments.view.summary=Gweld prisiad cryno
 label.viewAssessmentRadio=Prisiad
 
 assessments.localAuthorityRef=Cyfeirnod yr awdurdod lleol:
-assessments.property=Eiddo:
+assessments.caption=Eich eiddo
 assessments.title=Dewis prisiad i wirio
-assessments.pageTitle=Dewis prisiad i wirio
-assessments.paragraph=Mae gwerth ardrethol eiddo yn cael ei ailasesu gan y VOA bob tro y byddan nhw’n cyhoeddi rhestr ardrethi busnes newydd.  Gallwch wirio’r ffeithiau a’r gwerth ardrethol ar gyfer pob un o’r prisiadau a gyhoeddwyd.
+assessments.subtitle=Prisiadau ar gyfer yr eiddo hwn
 
-assessments.current.title=Y prisiad presennol
 assessments.draft.title=Y prisiad yn y dyfodol
 assessments.historic.title=Prisiadau blaenorol
-assessments.select.valuation=Dewiswch y prisiad hwn
-
 assessments.current.text=Presennol
 assessments.draft.text=Drafft
 assessments.historic.text=Hanesyddol
 assessments.aria.label.full=Dewiswch y prisiad hwn - {0} o {1} i {2}
 assessments.aria.label.partial=Dewiswch y prisiad hwn - {0} o {1}
+assessments.selectValuation.to=i
 
 assessments.na=Dim ar gael
+assessments.icon.help.valuationPeriod = Cymorth gyda Prisiadau
+assessments.caption.ip = Eich eiddo
+assessments.caption.agent = Eiddo''r cleient: {0}

```

Comment:
This is a bit of a nitpick but for apostrophes as a team we decided to use curly variant ’ and not the standard one on a keyboard ' to align with GDS standards. There may still be a few occurrences of ' variant in the message files.